### PR TITLE
Switch to sem place if intel RS service is available

### PR DESCRIPTION
**Pull request by KavitShah1998**

### CODE REVIEW

File: spot_rl_experiments/spot_rl/utils/skill_executor.py [hunk 1]
```diff
@@ -147,16 +147,20 @@ def execute_skills(self):
...
+                    skill_input,
+                    is_local=True,
+                    visualize=False,
+                    enable_waypoint_estimation=True,
+                )
+            else:
+                # Use the following for the hardcode waypoint for static place
+                succeded, msg = self.spotskillmanager.place(
+                    0.6, 0.0, 0.4, is_local=True
+                )
```
Comment (KavitShah1998) [inline spot_rl_experiments/spot_rl/utils/skill_executor.py:163]: This is the main change of this PR. Rest are clean-ups along the way

@jimmytyyang @tusharsangam

File: spot_rl_experiments/spot_rl/envs/skill_manager.py [hunk 2]
```diff
@@ -199,21 +209,24 @@ def __init_spot(
             else open_close_drawer_config
         )
 
-        self.open_close_drawer_config = (
-            construct_config_for_open_close_drawer()
-            if not open_close_drawer_config
-            else open_close_drawer_config
-        )
+        # self.open_close_drawer_config = (
```
Comment (jimmytyyang) [inline spot_rl_experiments/spot_rl/envs/skill_manager.py:212]: why do we want to remove this?

Comment (KavitShah1998) [reply spot_rl_experiments/spot_rl/envs/skill_manager.py:212]: It seems to be duplicate of code above it no?

Comment (jimmytyyang) [reply spot_rl_experiments/spot_rl/envs/skill_manager.py:212]: You are right, Kavit! Then remove these comments entirely please. Thanks in advance!

**Review by jimmytyyang — APPROVED** [review]
LGTM!

File: spot_rl_experiments/spot_rl/skills/atomic_skills.py [hunk 3]
```diff
@@ -389,8 +389,12 @@ def update_and_check_status(self, goal_dict: Dict[str, Any]) -> Tuple[bool, str]
...
             _nav_target_pose_deg,
-            self.config.SUCCESS_DISTANCE,
-            self.config.SUCCESS_ANGLE_DIST,
+            self.config.SUCCESS_DISTANCE_FOR_DYNAMIC_YAW_NAV
+            if self.env._enable_dynamic_yaw
+            else self.config.self.config.SUCCESS_DISTANCE,
+            self.config.SUCCESS_ANGLE_DIST_FOR_DYNAMIC_YAW_NAV
+            if self.env._enable_dynamic_yaw
+            else self.config.self.config.SUCCESS_ANGLE_DIST,
         )
```
Comment (KavitShah1998) [inline spot_rl_experiments/spot_rl/skills/atomic_skills.py:398]: `self.config.self.config` should be corrected to `self.config`

Comment (KavitShah1998) [reply spot_rl_experiments/spot_rl/skills/atomic_skills.py]: addressed

File: spot_rl_experiments/spot_rl/envs/skill_manager.py (hunk 2 above)
Comment (KavitShah1998) [reply spot_rl_experiments/spot_rl/envs/skill_manager.py]: This is addressed.

**Review by jimmytyyang — APPROVED** [review]
LGTm!

**Pull request merged**